sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -251,30 +231,52 @@ export default function CompReadyTestReport(props) {
       return itemB - itemA
     })
 
+    if (!versions[version]) {
+      // Handle the case where GA date is undefined (implies under development and not GA)
+      const weeksBefore = Math.floor(
+        (toDate - fromDate) / (1000 * 60 * 60 * 24 * 7)
+      )
```

Comment:
```
Sample (being evaluated) Release: 4.14
  Start Time: 2023-06-01T00:00:00Z
  End Time: 2023-06-07T23:59:59Z
  0 week(s) before 6/7/2023 of 4.14
```

If weeksBefore is 0 maybe we should show the days, or maybe just not anything at all?